core/core-impl/src/main/java/ai/timefold/solver/core/api/solver/SolutionManager.java

```diff
@@ -126,7 +126,7 @@ default ScoreAnalysis<Score_> analyze(Solution_ solution, ScoreAnalysisFetchPoli
      * Calculates and retrieves information about which constraints contributed to the solution's score.
      * This is a faster, JSON-friendly version of {@link #explain(Object)}.
      *
-     * @param solution never null, must be fully initialized otherwise an exception is thrown
+     * @param solution never null
```

Comment:
Note: this is technically a change to the public API, but because we're *removing* an exception, it is allowed; this change will not break anyone.